jQuery/README.md

```diff
@@ -8,12 +8,6 @@ Download the example and use NPM to install required libraries.
 npm install
```

Comment:
Change the title to `Getting Started with TagBox - jQuery`. GitHub won't let me.